torchtext/nn/modules/multiheadattention.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, dropout=0.0, batch_first=False):
                 as `(batch, seq, feature)`. Default: ``False``
 
         Examples::
-            >>> SDP = torchtext.nn.ScaledDotProduct(dropout=0.1)
+            >>> SDP = torchtext.nn.ScaledDotProduct(dropout=0.1, batch_first=True)
```

Comment:
In the example here, let's show the case without batch first.

Reply:
@zhangguanheng66 Fixed. Please review.